### ISSUE TITLE
Impossible to set up multiple availability slots on the same day

### ISSUE THREAD

**Issue by FlxMgdnz** [calcom/cal.com#12674]
### Issue Summary

When setting up my availability, I cannot specify e.g. Monday 9–11 am and 3–5 pm. When adding a second time slot, the dropdown for the From time is limited to 11:45 for the example given above.

![image](https://github.com/calcom/cal.com/assets/20115649/c2da2e50-e964-4301-92d7-858cd567d1e8)

### Technical details

Chrome 119 on Windows 11

**Comment by Tanujgarg37** [comment]
![image](https://github.com/calcom/cal.com/assets/87695633/ffb14761-6b09-46cc-b04f-35bfe1ef1f8c)
What type of error are u telling . Can you plz explain as I think I am not getting any error

**Comment by joeczucha** [comment]
I think they could only see until 11:45 because the second box had 12:00 in it.

You have a note about it in the yellow box here: https://cal.com/docs/how-to-guides/setting-up-your-availability

**Comment by PeerRich** [comment]
yeah you have to change the endtime. i wonder if we can come up with a better UI here. any ideas?

**Comment by raviakella03** [comment]
The time after the previous end time can be considered as start time of the next one.

For eg, first availability is 10:00 to 12:00 then the second availability of the day can start with 12:15 and the end time from 12:30.

**Comment by CarinaWolli** [comment]
This is expected behaviour
